tests/test_waterwheel.py

```diff
@@ -52,14 +58,19 @@ def test_common_names(self):
         self.assertEqual(str(doc.ents[0]), 'Is')
         self.assertEqual(str(doc.ents[1]), 'is')
         self.assertEqual(str(doc.ents[2]), 'IS')
+        for ent in doc.ents:
+            self.assertIsNotNone(ent._.wikilink)
     
     def test_issue20(self):
         doc = self.nlp('The Mackenzie River flows from the Great Slave Lake into the Arctic Ocean.')
-        self.assertEqual(str(doc.ents[1]), 'Great Slave Lake')
+        self.assertEqual(str(doc.ents[1]), 'Great Slave')
         self.assertEqual(str(doc.ents[1].label_), 'LAKE')
         doc = self.nlp('The Mackenzie River flows from the great slave lake into the Arctic Ocean.')
-        self.assertEqual(str(doc.ents[1]), 'great slave lake')
+        self.assertEqual(str(doc.ents[1]), 'great slave')
```

Comment:
same

Reply:
Updated PL with rule changes and test cases.